packages/node-dev/tsconfig.json

```diff
@@ -3,6 +3,7 @@
 	"compilerOptions": {
 		"outDir": "dist",
 		"preserveSymlinks": true,
+		"tsBuildInfoFile": "dist/typecheck.tsbuildinfo",
```

Comment:
But this is not only for typechecking right? It has an `outDir`

Reply:
`tsconfig.json` is only used for typechecking. all builds use `tsconfig.build.json`, where this property is overwritten.

Reply:
Sorry, I should've been clearer. `node-dev` has no `tsconfig.build.json` which means its `tsconfig.json` does double duty of typechecking and building, which means `dist/typecheck.tsbuildinfo` is not accurate. Just a passing thought.

Reply:
that makes sense. updated the branch 🙏🏽